portals/admin/site/public/locales/en.json

```diff
@@ -330,6 +331,10 @@
   "RolePermissions.TreeView.PermissionsSelector.scope.assignment.title": "Select Scope Assignments",
   "RolePermissions.TreeView.PermissionsSelector.update.scope.error": "Something went wrong while updating the permission",
   "ScopeAssignments.List.search.default": "Search by Role Name",
+  "Settings.Advanced.TenantConf.edit.success": "Tenant Configuration saved successfully",
```

Comment:
Is this the message we show once saved? Wondering why we have "Tenant" only here. In other places we have "Advanced" right?

Reply:
ahh, yes, will change to Advanced to be consistent with other places

Reply:
resolved